### PR TITLE
Minor: use straight quote marks in HTML fragments

### PR DESCRIPTION
**Pull request by frankieroberto**
(Curly quotation marks are invalid for HTML attributes)

### CODE REVIEW

**Review by 36degrees — APPROVED** [review]
Google Doc smart quotes strike again. Thanks Frankie 👍🏻 

This just needs a second review from the team and we can get this merged.

**Review by hannalaakso — APPROVED** [review]

**Review by calvin-lau-gds — APPROVED** [review]

**Pull request merged**
